STL_Extension/include/CGAL/Handle.h

```diff
@@ -46,8 +46,7 @@ class Handle
     Handle() noexcept
         : PTR(static_cast<Rep*>(0)) {}
 
-    // FIXME: if the precondition throws in a noexcept function, the program terminates
-    Handle(const Handle& x) noexcept
+    Handle(const Handle& x) noexcept(!(CGAL_preconditions || CGAL_assertions))
```

Comment:
Why is this test more restrictive than the one for `operator=`?

Reply:
This `CGAL_preconditions` is for the `CGAL_precondition`, and `CGAL_assertions` for the `CGAL_assume` which is a CGAL_assertion when assertions are enabled.

Reply:
Ah, right, github was misleadingly hiding the CGAL_assume, sorry.